views/question-table.pug

```diff
@@ -10,21 +10,28 @@ block styles
       .card-content
         h5.card-title Manage Questions
         h6 Add, modify or delete questions.
+        .switch
+          label
+            | Deleted
+            input(id='questionSwitch', type='checkbox', onClick='toggleQuestionSwitch()')
```

Comment:
In `toggleQuestionSwitch()` why don't you pass the parameter `this` and check whether it is checked or not in the function. Just in case an error happens this will make sure it is consistent